docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+services:
+  goose-cli:
+    build:
+      context: .
+      args:
+        USER_ID: ${UID:-1000}
+    volumes:
+      # Mount user's directory with read-write access
+      - .:/home/goose/workspace
+      # Mount git config (optional)
+      - ~/.gitconfig:/home/goose/.gitconfig:ro
+      # Mount SSH keys (optional)
+      - ~/.ssh:/home/goose/.ssh:ro
```

Comment:
how are these defined as optional? wouldn't this always mount the directories via docker compose?

Reply:
Yes, my goal was to inform users that the volume mounting configurations are adjustable - they can decide whether to comment out those lines. That seems to be confusing though. I removed the optional wording, as users can simply read the code and remove those lines if necessary.